misk/src/main/kotlin/misk/web/marshal/Generic.kt

```diff
@@ -91,13 +92,21 @@ object GenericMarshallers {
     }
   }
 
+  class ToUnit(private val contentType: MediaType?) : Marshaller<Unit> {
+    override fun contentType(): MediaType? = contentType
+    override fun responseBody(o: Nothing) = object : ResponseBody {
```

Comment:
```suggestion
    override fun responseBody(o: Unit) = object : ResponseBody {
```

I'm fairly confident that there's no way to pass `Nothing` as a parameter.